doc/changelog.md

```diff
@@ -157,78 +293,72 @@
   [{'expval0': (array(-0.0978434), array(-0.19767681))}, (array(-0.0978434), array(-0.19767681))]
   ```
 
-* Add native support for `qml.PSWAP` and `qml.ISWAP` gates on Amazon Braket devices. Specifically, a circuit like
-
-  ```py
-  import pennylane as qml
-  from catalyst import qjit
+* Support has been added for linear algebra functions which depend on computing the eigenvalues
+  of symmetric matrices, such as `np.sqrt_matrix()`.
+  [(#488)](https://github.com/PennyLaneAI/catalyst/pull/488)
 
-  dev = qml.device("braket.local.qubit", wires=2, shots=100)
+  For example, you can compile `qml.math.sqrt_matrix`:
 
-  @qjit
-  @qml.qnode(dev)
-  def f(x: float):
-      qml.Hadamard(0)
-      qml.PSWAP(x, wires=[0, 1])
-      qml.ISWAP(wires=[1, 0])
-      return qml.probs()
+  ```python
+  @qml.qjit
+  def workflow(A):
+      B = qml.math.sqrt_matrix(A)
+      return B @ A
   ```
 
-  would no longer decompose the `PSWAP` and `ISWAP` gates to `SWAP`s, `CNOT`s and `Hadamard`s. Instead it would just call Braket's native `PSWAP` and `ISWAP` gates at runtime.
-  [(#458)](https://github.com/PennyLaneAI/catalyst/pull/458)
-
-* Add support for the `BlockEncode` operator in Catalyst.
-  [(#483)](https://github.com/PennyLaneAI/catalyst/pull/483)
+  Internally, this involves support for lowering the eigenvectors/values computation lapack method
+  `lapack_dsyevd` via `stablehlo.custom_call`.
 
-* Remove copies of TOML device configuration files for Lightning device in Catalyst.
-  [(#472)](https://github.com/PennyLaneAI/catalyst/pull/472)
+* Additional debugging functions are now available in the `catalyst.debug` directory.
```

Comment:
I still think this could be considered a breaking change, these functions are not new, rather they have been moved from the `QJIT` class, which is part of the public API. (although they are better documented now)

Reply:
Okay, let me add an item here

Reply:
done